pgml-extension/requirements.linux.txt

```diff
@@ -89,85 +85,85 @@ nvidia-cufft-cu12==11.0.2.54
 nvidia-curand-cu12==10.3.2.106
 nvidia-cusolver-cu12==11.4.5.107
 nvidia-cusparse-cu12==12.1.0.106
-nvidia-ml-py==12.550.52
+nvidia-ml-py==12.555.43
 nvidia-nccl-cu12==2.20.5
-nvidia-nvjitlink-cu12==12.4.127
+nvidia-nvjitlink-cu12==12.5.82
 nvidia-nvtx-cu12==12.1.105
-openai==1.28.0
-optimum==1.19.2
-orjson==3.10.3
-outlines==0.0.34
-packaging==23.2
+openai==1.37.0
+optimum==1.21.2
+orjson==3.10.6
+outlines==0.0.46
+packaging==24.1
 pandas==2.2.2
-peft==0.10.0
-pillow==10.3.0
-plotly==5.22.0
-portalocker==2.8.2
+peft==0.11.1
+pillow==10.4.0
+plotly==5.23.0
+portalocker==2.10.1
 prometheus-fastapi-instrumentator==7.0.0
 prometheus_client==0.20.0
-protobuf==5.26.1
-psutil==5.9.8
+protobuf==5.27.2
+psutil==6.0.0
 py-cpuinfo==9.0.0
+pyairports==2.1.1
 pyarrow==11.0.0
 pyarrow-hotfix==0.6
-pydantic==2.7.1
-pydantic_core==2.18.2
+pycountry==24.6.1
+pydantic==2.8.2
+pydantic_core==2.20.1
 Pygments==2.18.0
-pynvml==11.5.0
 pyparsing==3.1.2
 python-dateutil==2.9.0.post0
 python-dotenv==1.0.1
 python-multipart==0.0.9
 pytz==2024.1
 PyYAML==6.0.1
-ray==2.21.0
+pyzmq==26.0.3
+ray==2.32.0
 referencing==0.35.1
-regex==2024.5.10
-requests==2.31.0
+regex==2024.5.15
+requests==2.32.3
 rich==13.7.1
 rouge==1.0.1
-rpds-py==0.18.1
+rpds-py==0.19.0
 sacrebleu==2.4.2
 sacremoses==0.1.1
 safetensors==0.4.3
-scikit-learn==1.4.2
-scipy==1.13.0
-sentence-transformers==2.7.0
+scikit-learn==1.5.1
+scipy==1.14.0
+sentence-transformers==3.0.1
 sentencepiece==0.2.0
 shellingham==1.5.4
 shtab==1.7.1
 six==1.16.0
 sniffio==1.3.1
-SQLAlchemy==2.0.30
+SQLAlchemy==2.0.31
 starlette==0.37.2
-sympy==1.12
+sympy==1.13.1
 tabulate==0.9.0
-tenacity==8.3.0
+tenacity==8.5.0
 threadpoolctl==3.5.0
-tiktoken==0.6.0
+tiktoken==0.7.0
 tokenizers==0.19.1
-torch==2.3.0
-torchaudio==2.3.0
-torchvision==0.18.0
+torch==2.3.1
+torchaudio==2.3.1
+torchvision==0.18.1
 tqdm==4.66.4
-transformers==4.40.2
+transformers==4.42.4
```

Comment:
We want `transformers==4.43.1` for the new Llama-3.1 models. See [here](https://huggingface.co/meta-llama/Meta-Llama-3.1-8B-Instruct#use-with-transformers).